File: app/src/main/java/com/lamnt/foodorder/model/dto/ResponseDTO.java
package com.lamnt.foodorder.model.dto;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.util.List;

public class ResponseDTO {
    @Expose
    @SerializedName("code")
    private String code;

    @Expose
    @SerializedName("message")
    private String message;

    @Expose
    @SerializedName("success")
    private String success;

    @Expose
    @SerializedName("data")
    private List<DemoEmployee> employee;

    public String getCode() {
        return code;
    }

    public void setCode(String code) {
        this.code = code;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }

    public String getSuccess() {
        return success;
    }

    public void setSuccess(String success) {
        this.success = success;
    }

    public List<DemoEmployee> getEmployee() {
        return employee;
    }

    public void setEmployee(List<DemoEmployee> employee) {
        this.employee = employee;
    }
}

File: app/src/main/java/com/lamnt/foodorder/model/dto/ResponseDTO.kt
package com.lamnt.foodorder.model.dto

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName
import java.io.Serializable

open class ResponseDTO : Serializable {
    var code: Int? = null

    @Expose
    @SerializedName("message")
    var message: String? = null

    @Expose
    @SerializedName("success")
    var success: Boolean? = null

    @Expose
    @SerializedName("status")
    var status: String? = null
}